app/routes/dashboard.apps.$appId.notifications/utils/notificationsAlertFormUtils.ts

```diff
@@ -0,0 +1,58 @@
+import { MantineTheme } from "@mantine/core"
+import { FREE_TIER_MAX_RELAYS } from "~/utils/pocketUtils"
+
+export type NotificationLevel = "quarter" | "half" | "threeQuarters" | "full"
+
+export const NOTIFICATIONS_ALERT_LEVELS: NotificationLevel[] = [
+  "quarter",
+  "half",
+  "threeQuarters",
+  "full",
+]
+
+export const DEFAULT_ALERT_PERCENTAGES = {
+  quarter: false,
+  half: false,
+  threeQuarters: true,
+  full: true,
+}
+
+export function getUsagePercentage(usageLevel: string): string {
```

Comment:
Can't we follow the same approach here as we're doing with `DEFAULT_ALERT_PERCENTAGES `

instead of a function use a constant and map the values since they're pre-defined. Simpler and less if statements.

Reply:
Will do!